data/hai/hai ships.txt

```diff
@@ -514,6 +595,22 @@ ship "Lightning Bug"
 	"final explode" "final explosion small"
 	description "The Lightning Bug is a small defensive ship, mostly used by the Hai as a convoy escort. The Lightning Bug, as with many insects, works best in a swarm: even a small number of Lightning Bugs can manage to leave the largest of ships helpless with their stock-fitted Ion Cannons."
 
+
+ship "Lightning Bug" "Lightning Bug (Ionic)"
```

Comment:
There's 1 turret but I see what you mean

I dont think its necessary to fill all hardpoints, and the sea scorp isnt a garanteed spawn iirc
This turret is in fact the biggest threat in this fight

Reply:
Lightning Bugs have one gun mount and two turret mounts.

Reply:
I added a bullfrog to it instead of removing the turret as I feel like a lightning bug with just a blaster (which is just a glorified pulse cannon, at 19 space) isn't that relevant in battle, and I want this variant to be more generic than just for this encounter.
The point could be made to make it weak if the fight is estimated to be too hard, of course